Misc/NEWS.d/next/Library/2019-07-20-01-17-43.bpo-36161.Fzf-f9.rst

```diff
@@ -0,0 +1 @@
+Use thread-safe functions instead of unsafe ones (crypt, ttyname)
```

Comment:
"Use ttyname_r instead of ttyname and crypt_r instead of crypt for thread safety."

Reply:
I forgot to update this, because it was autogenerated from the first commit. Thanks a lot for your review :wink: